### ISSUE TITLE
Bug Mode?

### ISSUE THREAD

**Issue by w4mmp-ron** [n1gp/iambic-keyer#2]
Hello,
I was fortunate and found your Iambic-keyer routine.  It has been adapted to run in a Microchip MCU (PIC16F326).  After addressing timing issues due to the slower processing time of the MCU the routine is running very well.   However the readme file indicates the routine has BUG mode but I can't find that in the code.   What am I missing?

Regards,
Ron
W4MMP

**Comment by n1gp** [comment]
The code was copied/translated from a verilog project. Those comments are not mine.
But I think what it is saying is that bug mode is the same as straight key. The multiple
dots are input from a mechanical device, not in the code.

**Comment by w4mmp-ron** [comment]
Yes, after thinking about it a bit more, a user would put the mode to straight for a bug.

Thanks again for the code.  It is working well on the Microchip MCU.
